osm2lanes/src/transform/tags_to_lanes/road.rs

```diff
@@ -148,13 +158,22 @@ impl RoadBuilder {
                 None
             },
         };
+
+        let width = locale.travel_width(&designated, highway.r#type());
+        let width = || Width {
+            min: Infer::None,
+            target: Infer::Default(width),
+            max: Infer::None,
+        };
```

Comment:
is using a closure better than a clone? unclear.

Reply:
A clone would be a bit more readable, IMO. If we keep the closure, could we name the first variable 'width_value'? Shadowing is a feature I wish Rust didn't have

Reply:
I don't really mind variable shadowing, I can't remember a time where I made mistakes because of it.
I do mainly shadow the variable with a different type, so the compiler catches most problems.

Reply:
I will change it to a clone then.